runtime/compiler/compile/J9Method.cpp

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
```

Comment:
Should the copyright on these new files be 2019, 2019?

Reply:
Yes and no.  Technically you are correct that this is a new file.  But very soon it will be the home to a bunch of functions from OMR that have been around since 2000.  So, I just kept the copyright as it is in omr/compiler/compile/Method.cpp (actually, since I discovered the copyright in OMRMethod.cpp is wrong because the initial copyright should be 2000, not 2019--I'll fix that).  I don't mind changing it if that's what you want, and then reset it again when the functions actually move here.

Reply:
Thanks for the explanation. No need to change given the history and direction here.